src/fields/fieldText.vue

```diff
@@ -11,7 +11,7 @@
 	
 </script>
 
-<style lang="sass" scoped>
+<style lang="sass">
 	input {
```

Comment:
It will need something like `.vue-form-generator .field-*** input`, because some fields don't look good when stretched to 100% (color for example).